doc/changelog/07-vernac-commands-and-options/14940-rm-output-state-for-good.rst

```diff
@@ -0,0 +1,5 @@
+- **Removed:**
+  the `-outputstate` command line argument and the corresponding
+  vernacular commands `Write State` and `Restore State`
```

Comment:
@coq/doc-maintainers is a single backquote correct or do we want doubled backquotes here? I don't remember the difference.

Reply:
Single backquotes can be mapped to many things in Sphinx. But IIRC, we've mapped them to be the same as double backquotes. So this should be fine.